rbac/management/role/view.py

```diff
@@ -639,6 +641,20 @@ def validate_role(self, request):
                         error = {key: [_(message)]}
                         raise serializers.ValidationError(error)
 
+                # validate permission not being added to workspace out of users org for v1 (RHCLOUD-35481)
+                resourceDefinitions = perm.get("resourceDefinitions")
```

Comment:
I think we do explicitly require `resourceDefinitions` on the payload, but would it be worth adding a default here so the `for` loop below is guarded?
```suggestion
                resourceDefinitions = perm.get("resourceDefinitions", [])
```

Reply:
- [ ] add default for list